openmdao/utils/tests/test_options_dictionary.py

```diff
@@ -32,41 +32,71 @@ class MyComp(ExplicitComponent):
+            "=========  ============  ===================  =====================  ====================",
         ]))
 
         # if the table can't be represented in specified width, then we get the full width version
         self.assertEqual(self.dict.__str__(width=40), '\n'.join([
-            "========= ============ ================= ===================== ====================="
-            "==================================================================== ",
-            "Option    Default      Acceptable Values Acceptable Types      Description          "
-            "                                                                     ",
-            "========= ============ ================= ===================== ====================="
-            "==================================================================== ",
-            "comp      MyComp       N/A               ['ExplicitComponent']                      "
-            "                                                                     ",
-            "flag      False        [True, False]     ['bool']                                   "
-            "                                                                     ",
-            "long_desc **Required** N/A               ['str']               This description is l"
-            "ong and verbose, so it takes up multiple lines in the options table. ",
-            "test      **Required** ['a', 'b']        N/A                   Test integer value   "
-            "                                                                     ",
-            "========= ============ ================= ===================== ====================="
-            "==================================================================== ",
+            "=========  ============  ===================  =====================  ====================="
+            "====================================================================",
+            "Option     Default       Acceptable Values    Acceptable Types       Description",
+            "=========  ============  ===================  =====================  ====================="
+            "====================================================================",
+            "comp       MyComp        N/A                  ['ExplicitComponent']",
+            "flag       False         [True, False]        ['bool']",
+            "long_desc  **Required**  N/A                  ['str']                This description is l"
+            "ong and verbose, so it takes up multiple lines in the options table.",
+            "test       **Required**  ['a', 'b']           N/A                    Test integer value",
+            "=========  ============  ===================  =====================  ====================="
+            "====================================================================",
         ]))
 
+    def test_to_table(self):
```

Comment:
I think we need a `skipTest` on this in case tabulate is not installed...  and/or a test for the "tabulate not installed" message...